source/coordinates.md

```diff
@@ -253,14 +254,14 @@ the order {math}`(x, y, z)`. {math}`\bf W` can be expressed in terms of the
 orientation angles {math}`\theta`, {math}`\phi`, and {math}`\psi` via the formula
 ```{math}
 :label: www
-  {\bf W} &= {\bf R}_{y}(\theta) \; {\bf R}_{x}(-\phi) \; {\bf R}_{z}(\psi) \\
+  {\bf W} &= {\bf R}_{y}(\theta) \; {\bf R}_{x}(\phi) \; {\bf R}_{z}(\psi) \\
   &= \begin{pmatrix}
-    \cos\theta \cos\psi - \sin\theta \sin\phi \sin\psi & 
-        -\cos\theta \sin\psi - \sin\theta \sin\phi \cos\psi & 
+    \cos\theta \cos\psi + \sin\theta \sin\phi \sin\psi & 
+        -\cos\theta \sin\psi + \sin\theta \sin\phi \cos\psi & 
          \sin\theta \cos\phi \\
-    \cos\phi \sin\psi & \cos\phi \cos\psi & \sin\phi \\
-   -\cos\theta \sin\phi \sin\psi - \sin\theta \cos\psi & 
-         \sin\theta \sin\psi - \cos\theta \sin\phi \cos\psi & 
+    \cos\phi \sin\psi & \cos\phi \cos\psi & -\sin\phi \\
+   \cos\theta \sin\phi \sin\psi - \sin\theta \cos\psi & 
+         \sin\theta \sin\psi + \cos\theta \sin\phi \cos\psi & 
```

Comment:
It would be good to double-check the sign changes here.  (I can take a look shortly.)